eng/pipelines/runtime-staging.yml

```diff
@@ -102,6 +103,7 @@ jobs:
     runtimeFlavor: mono
     platforms:
     - Android_arm64
+    - Android_arm
```

Comment:
Are there Android arm devices available yet? It seems like this uses the same helix queue as arm64: https://github.com/dotnet/runtime/blob/944805ef701bfae514f5216b893b03292af33701/eng/pipelines/libraries/helix-queues-setup.yml#L106

Reply:
There aren't.  I made the mistake thinking that there were.  We are just building arm at the moment.